### PR TITLE
Performance tests workers and workflows

### PR DESCRIPTION
**Pull request by Jozefiel**

### CODE REVIEW

File: frinx/workers/test/test_worker.py
Comment (dav1d-b) [inline frinx/workers/test/test_worker.py:1]: These tests don't contain any assert statemets. What are we really testing here?

Comment (Jozefiel) [reply frinx/workers/test/test_worker.py:1]: This collection of workers and workflows is used for performance tests.
Can be executed as a system test like in the old repository.

File: frinx/workers/test/test_worker.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            output: str
+
+        def execute(self, worker_input: WorkerInput) -> TaskResult:
+            task_result = TaskResult(
+                status=TaskResultStatus.COMPLETED, logs=['Echo worker invoked successfully']
+            )
+            task_result.add_output_data('output', worker_input.input)
+            return task_result
+
+    ###############################################################################
```
Comment (dav1d-b) [inline frinx/workers/test/test_worker.py]:
```suggestion
```

File: frinx/workers/test/test_worker.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            import time
+
+            task_result = TaskResult(
+                status=TaskResultStatus.COMPLETED, logs=['Sleep worker invoked. Sleeping']
+            )
+            time.sleep(sleep)
+            task_result.add_output_data('time', sleep)
+            return task_result
+
+    ###############################################################################
```
Comment (dav1d-b) [inline frinx/workers/test/test_worker.py]:
```suggestion
```

File: frinx/workers/test/test_worker.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+        def execute(self, worker_input: WorkerInput) -> TaskResult:
+            sleep = worker_input.time if worker_input.time else self.DEFAULT_SLEEP
+            if sleep < 0 or sleep > self.MAX_SLEEP:
+                return TaskResult(
+                    status=TaskResultStatus.FAILED,
+                    logs=['Invalid sleep time, must be > 0 and < 600'],
+                )
+
+            import time
```
Comment (dav1d-b) [inline frinx/workers/test/test_worker.py]: Move it to the beginning of the file.

File: frinx/workers/test/test_worker.py [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+            task_result = TaskResult(
+                status=TaskResultStatus.COMPLETED, logs=['Sleep worker invoked. Sleeping']
+            )
+            time.sleep(sleep)
+            task_result.add_output_data('time', sleep)
+            return task_result
```
Comment (dav1d-b) [inline frinx/workers/test/test_worker.py]: Just a suggestion. I find this style more readable.

```suggestion
            time.sleep(sleep)
            return TaskResult(
                status=TaskResultStatus.COMPLETED,
                logs=['Sleep worker invoked. Sleeping'],
                output={'time': sleep}
            )
```

**Review by dav1d-b — CHANGES_REQUESTED** [review]

File: frinx/workers/test/test_worker.py [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+        class WorkerOutput(TaskOutput):
+            output: str
+
+        def execute(self, worker_input: WorkerInput) -> TaskResult:
+            task_result = TaskResult(
+                status=TaskResultStatus.COMPLETED, logs=['Echo worker invoked successfully']
+            )
+            task_result.add_output_data('output', worker_input.input)
+            return task_result
```
Comment (dav1d-b) [inline frinx/workers/test/test_worker.py]:
```suggestion
            return task_result = TaskResult(status=TaskResultStatus.COMPLETED, logs=['Echo worker invoked successfully'], output={'output': worker_input.input})
```

**Review by dav1d-b — APPROVED** [review]

**Pull request merged**